src/getting-started-postman-collection.handlebars

```diff
@@ -146,13 +146,13 @@
                     <div class="col-xs-12">
                       <p>In the <code>Akeneo PIM API</code> collection, select the request named <code>Authentication by password</code>.</p>
                       <img class="img-responsive img-thumbnail center-block" src="/img/authentication-request-in-postman.png"><br>
-                      <p>Then, hit the <code>Send</code> blue button.</p>
+                      <p>Then, hit the blue <code>Send</code> button.</p>
                       <img class="img-responsive img-thumbnail" src="/img/send-button.png"><br>
-                      <p>In the response area, you should get an answer similar to this one below.</p>
+                      <p>In the response area, you should get an answer similar to the one below.</p>
                       <img class="img-responsive img-thumbnail" src="/img/authentication-response.png">
-                      <div class="alert alert-warning">The authentication token only lasts one hour. This means that, after one hour, your token will expire. And you will receive a error message that looks like this.
+                      <div class="alert alert-warning">The authentication token only lasts for an hour. This means that after one hour your token will expire. And you will receive an error message that looks like this.
```

Comment:
I would keep it as one hour because it's really "60 minutes". But I trust you on the english part :)